src/pspm_get_sps.m

```diff
@@ -0,0 +1,43 @@
+function [sts, data] = pspm_get_sps(import, eye)

```

Comment:
I think we should stick to the usual convention of PsPM mainly `pspm_get_any_FILE_type` should take only 'datafile' and 'import' as arguments (e.g. `pspm_get_edf`, `pspm_get_biotrace`, `pspm_get_eyelink`, etc.) and `pspm_get_any_DATA_type` should only take 'import' as argument (e.g. `pspm_get_emg`, `pspm_get_ecg`, `pspm_get_custom`, etc.). In the case of data types related to eyes usually we have two distinct function, e.g `pspm_get_blink_l` and `pspm_get_blink_r`. So I would say that you should separate this function into `pspm_get_sps_l` and `pspm_get_sps_r`.

Reply:
makes sense, have updated